indico/modules/rb_new/client/js/components/BookingTimeInformation.jsx

```diff
@@ -0,0 +1,73 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Icon, Segment, Label} from 'semantic-ui-react';
+import {Translate, Param} from 'indico/react/i18n';
+import {toMoment} from 'indico/utils/date';
+import recurrenceRenderer from './RecurrenceRenderer';
+
+import './BookingTimeInformation.module.scss';
+
+
+export default function BookingTimeInformation({recurrence, dates, timeSlot}) {
+    const {startDate, endDate} = dates;
+    const {startTime, endTime} = timeSlot;
+    const {type} = recurrence;
+    const sDate = toMoment(startDate);
```

Comment:
i don't like the idea of mixing sDate and startDate for moment/string versions...

Reply:
I actually just extracted this to standalone component and didn't look much at the logic inside, but yeah, it's a good opportunity to change it :)